packages/table/components/Table.tsx

```diff
@@ -285,19 +261,7 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
   }
 
   private cellRenderer(args: GridCellProps) {
-    const { columnIndex, rowIndex, key, parent } = args;
-
-    return (
-      <CellMeasurer
-        cache={this.cellMeasureCache}
-        columnIndex={columnIndex}
-        key={key}
-        parent={parent}
-        rowIndex={rowIndex}
-      >
-        {this.getCell(args)}
-      </CellMeasurer>
-    );
+    return this.getCell(args);
```

Comment:
am I correct that this change breaks existing functionality? If so, please add a notice to commit message 🙇 

e.g.
```
fix(table): remove content-based column width sizing

BREAKING CHANGE:
This change <whatitdoes>

Before: <short explanation>
After: <short explanation>

Closes DCOS-50093
```

going forward we should take care of breaking changes and our commit messages in general - and especially in ui-kit, since we're going to release dcos-ui quite often and need to be able to maintain older versions. 
remember the issues we had with `reactjs-components` 😿 

ref: https://www.conventionalcommits.org/en/v1.0.0-beta.3/#specification

Reply:
Ok, I was undecided whether to add a breaking change for this commit since everything will still build and run just fine.
But I guess the fact that table columns without `width` passed won't behave the same as they used to is a good enough reason to flag it as a breaking change 👍